### PR TITLE
Adding React Charting Library VR Tests

### PR DESCRIPTION
**Pull request by yush-singla**
## Description
This PR integrates the React Charting Library to the vr-tests application to enable it for vr testing. 

### Idea
We Plan to have about 150 tests/screenshots added to the v8 application (vr-tests) for testing of the react-charting library.

### Performance Implication
- It would increase the build time for the v8 visual regression test pipeline by about 1-2 minutes when the pipeline runs.
- This pipeline will be triggered whenever there's a change in the @fluentui/react-charting package in addition to the current triggers. 
- The percentage of PRs related to react-charting package in github are less than 10% of the total pull request to the repository.

### CODE REVIEW

File: apps/vr-tests/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@fluentui/react-hooks": "*",
```
Comment (fabricteam) [inline apps/vr-tests/package.json:27]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28634&commitId=2067ff29dda7d035846688c2657938a35d37feeb&env=prod&clientType=FLUENTUI)


  <details><summary><b>PieChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PieChart.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,967 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../../utilities/index';
+import { IChartProps } from '../../../../../packages/react-charting/src/HorizontalBarChart';
```
Comment (AtishayMsft) [inline apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx:5]: why using relative paths and not @fluentui/react-charting.
Similarly for other imports also.

File: apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+    <StoryWright
+      steps={new Steps()
+        .snapshot('default', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>,
+  )
+  .addStory(
+    'Root',
```
Comment (AtishayMsft) [inline apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx:21]: Use the same names as in react-examples.

File: apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,597 @@
...
+              ) : null
+            }
+          />
+        </div>
+      );
+    },
+    { includeDarkMode: true, includeRtl: true },
+  )
+  .addStory(
+    'Styled',
```
Comment (AtishayMsft) [inline apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx:158]: we can remove area chart styled. Does not serve anything new.

File: apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,597 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { StoryWright, Steps } from 'storywright';
+import { TestWrapperDecorator } from '../../utilities/TestWrapperDecorator';
+import { AreaChart, ICustomizedCalloutData, ChartHoverCard } from '@fluentui/react-charting';
+import { DefaultPalette } from '@fluentui/react';
```
Comment (AtishayMsft) [inline apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx:6]: Can we add 1 more screenshot for following:
Hover over basic area chart, hover over multiple area chart.

File: apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,967 @@
...
+              enablePerfOptimization={true}
+            />
+          </div>
+        </>
+      );
+    },
+    { includeRtl: true, includeDarkMode: true },
+  )
+  .addStory(
+    'Date_Axis',
```
Comment (AtishayMsft) [inline apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx:834]: We can remove date axis story and add story to test - hover card for single line.

**Review by AtishayMsft — APPROVED** [review]

**Review by TristanWatanabe — APPROVED** [review]

**Pull request merged**